.gitignore

```diff
@@ -6,4 +6,5 @@ hc.sqlite
 hc/local_settings.py
 static-collected
 .DS_Store
-static/.DS_Store
\ No newline at end of file
+static/.DS_Store
+.vscode
```

Comment:
System generated files like `.DS_Store` and editor files like `.vscode` don't need to be added to the gitignore file since different people use different systems/editors and if everyone added their files the .gitignoew file would be bloated. Please move them to the `.gitignore_global` file

You also need to add a blank like at the end of your file

Reply:
We've removed system-generated and editor files and added them to a global gitignore file instead.
[Commit 277e235](https://github.com/andela/hc-june-bunnies/pull/5/commits/277e23531ba8ead4b0887dc65335bd75686469ce)